src/scene_battle_rpg2k.cpp

```diff
@@ -102,9 +102,7 @@ void Scene_Battle_Rpg2k::CreateBattleCommandWindow() {
 }
 
 void Scene_Battle_Rpg2k::RefreshCommandWindow() {
-	std::string skill_name = active_actor->GetSkillName();
-	command_window->SetItemText(1,
```

Comment:
> 2.4: RM2000 (not tested in RM2003): If you have the "special skill name...

Style question: As this is only used in combination with GetSkill what do you think about moving this "IsSkillRenamed" logic directly in "GetSkillName"?

Reply:
I think that makes sense. From Player perspective there is only 1 skill name. The renaming stuff is a liblcf detail.